src/styles/dialog.less

```diff
@@ -34,6 +34,12 @@ ai-dialog-container {
   &.active {
     opacity: 1;
   }
+  
+  & > div {
+    // Prevent Chrome on Windows from adding a focus outline. For details, see
+    // https://github.com/twbs/bootstrap/pull/10951
+    outline: 0;
```

Comment:
@Gheoan yes it is necessary
the parent one it's not i just put it for extra measure
![2016-02-26_13h54_24](https://cloud.githubusercontent.com/assets/1510389/13342757/bcfba1ec-dc90-11e5-9ff4-95aded27c252.png)